### DISCUSSION TITLE
Kami 9FUS firmware downgrade.

### DISCUSSION THREAD

**Discussion by Vicktor1st** [roleoroleo/yi-hack-MStar#517] [General]
My friend gave me two cameras Kami 9FUS but they have firmware 4.7 :(
Is there any way to downgrade firmware in this camera?
Force downgrade by external programmer?

**Comment by roleoroleo** [comment]
I don't know this firmware and probably it's not a MStar cam.
But AFAIK there is no mode to backup the fw without a serial connection (or external spi programmer).
If you are able to backup the fw, I can check which model is.
